docs/logging-and-user-files.md

```diff
@@ -8,8 +8,6 @@ This section describes the user-accessible files used by the Ping Viewer applica
 
 Ping Viewer will create a folder called `PingViewer` in your user documents folder:
 
-TODO use full path
```

Comment:
Why are we removing this ? We are not using full, but relative paths.

Reply:
Is there a good reason to use full paths here? I don't know about macOs, but with Windows and Linux it is actually harder to get to these directories using the full path.

Reply:
> Is there a good reason to use full paths here?

no the todo is moot